src/main/resources/doca.properties

```diff
@@ -1,7 +1,11 @@
 #Fields that need to be anonymized in doca
-doca_fields=kwh
+doca_fields=Seconds_EnergyConsumption
```

Comment:
please rebase on current master and test again, the field name that you use here is from an outdated dataset

Reply:
(test with current dataset)

Reply:
Tested. Works with the current dataset